src/main/java/wooteco/subway/station/ui/StationController.java

```diff
@@ -0,0 +1,65 @@
+package wooteco.subway.station.ui;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.station.domain.Station;
+import wooteco.subway.station.service.StationService;
+import wooteco.subway.station.ui.dto.StationRequest;
+import wooteco.subway.station.ui.dto.StationResponse;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class StationController {
+
+    private final StationService stationService;
+
+    public StationController(final StationService stationService) {
+        this.stationService = stationService;
+    }
+
+    @PostMapping("/stations")
+    public ResponseEntity<StationResponse> createStation(@RequestBody StationRequest stationRequest) {
+        Station station = new Station(stationRequest.getName());
+        Station newStation = stationService.createStation(station);
+        StationResponse stationResponse = new StationResponse(newStation.getId(), newStation.getName());
+
+        return ResponseEntity.created(
+                URI.create("/stations/" + newStation.getId())
+        ).body(stationResponse);
+    }
+
+    @GetMapping(value = "/stations", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<StationResponse>> showStations() {
+        List<Station> stations = stationService.findAll();
+
+        List<StationResponse> stationResponses = stations.stream()
+                .map(it -> new StationResponse(it.getId(), it.getName()))
+                .collect(Collectors.toList());
+
+        return ResponseEntity.ok().body(stationResponses);
+    }
+
+    @DeleteMapping("/stations/{id}")
+    @SuppressWarnings({"rawtypes"})
+    public ResponseEntity deleteStation(@PathVariable Long id) {
```

Comment:
아래처럼 하면 `@SuppressWarnings({"rawtypes"})`를 사용하지 않아도 됩니다 :) 

```suggestion
    public ResponseEntity<Void> deleteStation(@PathVariable Long id) {
```

Reply:
Void가 있는건 처음 알았네요..! 신기해요... 감사합니다!!!